django_project/feti/views/api.py

```diff
@@ -33,6 +33,7 @@
 __copyright__ = 'kartoza.com'
 
 
+# this is not used
```

Comment:
please remove this comment (although the API is not used, no need the comment)

Reply:
I thought it could help someone better understand the code they are reading.